claripy/strings.py

```diff
@@ -2,31 +2,36 @@
 from .backend_object import BackendObject
 from .bv import BVV
 
+
 class StringV(BackendObject):
     def __init__(self, value):
         self.value = value
 
     def __repr__(self):
         return 'StringV(%s)' % (self.value)
 
+
 def StrConcat(*args):
     """
-    Create a concrete version of the concatenated string
-    :param args: List of string that has to be concatenated
+    Create a concrete version of the concatenated string.
```

Comment:
...agh. I wish I had reviewed the PR that added string more carefully. All of these functions don't create concrete anything, they are merely the operations implemented for the concrete String objects.